components/ManagerSubmitButton.tsx

```diff
@@ -0,0 +1,54 @@
+import urlsForRouter from "helpers/urlsForRouter";
+import { Button } from "lbh-frontend-react";
+import { NextRouter, useRouter } from "next/router";
+import PropTypes from "prop-types";
+import React from "react";
+import PageSlugs, { urlObjectForSlug } from "steps/PageSlugs";
+
+interface Props {
+  onSubmit(router: NextRouter): Promise<void>;
+  buttonText: string;
+}
+
+const ManagerSubmitButton: React.FunctionComponent<Props> = (props) => {
+  const { onSubmit, buttonText } = props;
+  const router = useRouter();
+  const status = buttonText === "Approve" ? "approved" : "declined";
+
+  const { href, as } = urlsForRouter(router, {
+    ...urlObjectForSlug(router, PageSlugs.Confirmed),
```

Comment:
Should this go to a new `ManagerSubmit` page? We don't want to persist any actions if they're offline (which is technically possible on this page). That's what the `Submit` and `Confirmed` pair of pages are for in the normal process, and I think we need the same here.

Reply:
On second thought, I think this is actually right, as long as the manager can't make any changes to the process itself. We should probably adjust the loading page text to assume that a manager is online, though, even if we handle them being offline, as we expect them to be reviewing from their desktops.